pr_request_templete.md

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
Can you put this in the `.github` directory and use UPPER_SNAKE_CASE? This is a common convention e.g. https://github.com/facebook/react/tree/main/.github.

Reply:
Hey I had made the required changes can you have a look at it?

Reply:
I see that you renamed and moved the template. That looks great.

I think that we can organize the template better though. Please use [the example that I suggested](https://github.com/sparkbox/sparkeats/pull/397#pullrequestreview-1135121428) as a starting point or make the way you have organized it.

Reply:
should I copy paste the contribution guide as you have given and add some more thing's in it? will it be fine?

Reply:
Yes, I think that is a good starting point because it has only two minimal sections and it is not too prescriptive about what the reviewer needs to do.

- `What changed` for a description of the changes
- `How to validate` to explain to the reviewer how to make sure that the changes are correct.

I like the idea of the `Screenshots` section that you included, but I think screenshots can also be included as a part of `What changed`. We could even add a comment such as `
Include relevant screenshots or videos.`

Reply:
hmm looking great to me as well let me change the file...

Reply:
@yosevu can have a look at it?